packages/onedump.vm/onedump.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>onedump.vm</id>
```

Comment:
@mandiant/flare-vm what about creating `didier-stevens-suite.beta.vm` instead and creating the package in a very similar way to https://github.com/mandiant/VM-Packages/blob/fb715d6772ed61ba485117d46731d5aba96dcb7a/packages/didier-stevens-suite.vm/tools/chocolateyuninstall.ps1#L6 (with a shortcut to `onedump`)? Or we could even add the beta repository in the installation of `didier-stevens-suite.vm` as a single package. @mandiant/flare-vm @d35ha preferences?

Reply:
Combining both repositories in a single package is more convenient (with the priority to the scripts from DidierStevensSuite for duplicates).
But I'm not sure if creating a shortcut to a python script is a good idea, the interpreter won't be able to resolve the script location.
A better solution is to add `VM-Install-Single-Py` (just like [`VM-Install-Single-Ps1`](https://github.com/d35ha/VM-Packages/blob/main/packages/common.vm/tools/vm.common/vm.common.psm1#L399)) with the interpreter being executed upon executing the shortcut.

Reply:
I'll mutate `didier-stevens-suite.vm` for `DidierStevens/Beta`.